packages/@apphosting/adapter-nextjs/src/bin/build.ts

```diff
@@ -2,12 +2,58 @@
 import { spawnSync } from "child_process";
 import { loadConfig, readRoutesManifest } from "../utils.js";
 
+import { join } from "path";
+import fsExtra from "fs-extra";
+import { stringify as yamlStringify } from "yaml";
+
+const { move, exists, writeFile, mkdirp } = fsExtra;
```

Comment:
nit: I'd set a comment about fsExtra being CJS and not being able to use shorthand imports.